packages-next/auth/src/pages/SigninPage.tsx

```diff
@@ -101,21 +101,31 @@ export const SigninPage = ({
           </Notice>
         )}
         <Stack gap="medium">
+          <VisuallyHidden as="label" htmlFor="identity">
+            Email Address
```

Comment:
This won't always be an email address. We should use whatever the identify field is called here.